--- makefile	2014-10-04 15:48:43.000000000 +0200
+++ makefile.new	2016-02-05 22:07:18.214710576 +0100
@@ -98,11 +98,11 @@
 #setprec.o: setprelf.387
 #	$(AS) -o setprec.o setprelf.387
 
-sqrt.o: sqrtelf.amd64
-	$(AS) -o sqrt.o sqrtelf.amd64
+#sqrt.o: sqrtelf.amd64
+#	$(AS) -o sqrt.o sqrtelf.amd64
 
-setprec.o: setprelf.amd64
-	$(AS) -o setprec.o setprelf.amd64
+#setprec.o: setprelf.amd64
+#	$(AS) -o setprec.o setprelf.amd64
 
 # Motorola 68881. Caution, subroutine return conventions vary.
 #sqrt.o: sqrt.688
